src/libraries/System.Private.CoreLib/src/System/Globalization/Normalization.Icu.cs

```diff
@@ -40,10 +41,43 @@ private static unsafe bool IcuIsNormalized(string strInput, NormalizationForm no
             return ret == 1;
         }
 
+        private static unsafe bool IcuIsNormalized(ReadOnlySpan<char> source, NormalizationForm normalizationForm)
```

Comment:
This looks like a copy of the above method, just with span instead of string? Can we instead just change the above method to be for span?